src/components/HomePage.js

```diff
@@ -5,64 +5,79 @@ const HomePage = () => {
+                  <div className="col-lg-12 align-center" id="lead">
+                    <h1>Hoppi Space</h1>
+                    <h2>Connecting the artist within to the space you are without</h2>
+                  {/*TO-DO: Add Anchor linked buttons to content below*/}
+                  <form>
+                      <div class="col-lg-12">
+                        <div class="input-group">
+                          <input type="text" class="form-control" placeholder="Search for..." />
+                          <span class="input-group-btn">
+                            <button class="btn btn-secondary" type="button">Go!</button>
+                          </span>
+                        </div>
+                      </div>
+                    </form>
+                  </div>
+                </div>
               </div>
 
-              <section>
+            {/*Featured Studios Section*/}  
+              <div className="container">
                 <div id="featured-carousel">
                   <h2>Featured Studios</h2>
                   <h3>A selection of our favorite digs.</h3>
                   {/*TO DO: Implement Featured Studios carousel*/}
                 </div>
-              </section>
-
-            <section>
-              <div id="how-it-works">
-            {/*TO DO: Ad back countraction to "you've"*/}
-                <h2>How It Works</h2>
-                <h3>Browse, Book, Pay</h3>
-                <p>We are your new favorite local coffee shop community board. Browse hundreds of available workshops in your area or worldwide using our fast and efficient search engine. 
-                We optimize results with your craft needs, tools, and budget in mind. 
-                When youve found "the one", our effortless booking system will relay your request to our owners for confirmation, and utilizing our secure Braintree-based payment system, make your payment. Welcome home.</p>
-              </div>
-            </section>
-            <footer id="footer-items">
-              <div>
-                <h4>Socialize</h4>
-                <ul className="social-share" id="footer-social">
-                  <li>Facebook</li>
-                  <li>Instagram</li>
-                  <li>Twitter</li>
-                </ul>
               </div>
 
-              <div id="footer-contact">
-                <h4>Get In Touch</h4>
-                <ul>
-                  <li>General: contact@hoppispace.com</li>
-                  <li>Support: support@hoppispace.com</li>
-                  <li>{/*TO DO: Add phone icon*/}Phone: 000.000.0000</li>
-                </ul>
+            {/*How it Works Section*/}
+              <div className="container">
+                <div className="jumbotron" id="how-it-works">
+                {/*TO DO: Ad back countraction to "you've"*/}
+                  <h2 className="display-3">How It Works</h2>
+                  <h3>Browse, Book, Pay</h3>
+                  <hr/>
```

Comment:
Meant to be <br />?

Reply:
Oops it did an actual break. Should this be a br tag?

Reply:
This is a horizontal rule. That's expected 👍